src/main/scala/no/ndla/draftapi/service/StateTransitionRules.scala

```diff
@@ -164,15 +172,15 @@ trait StateTransitionRules {
     private[service] def doTransitionWithoutSideEffect(current: domain.Article,
                                                        to: ArticleStatus.Value,
                                                        user: UserInfo,
-                                                       isImported: Boolean): (Try[domain.Article], SideEffect) = {
+                                                       isImported: Boolean): (Try[domain.Article], Set[SideEffect]) = {
       getTransition(current.status.current, to, user) match {
         case Some(t) =>
           val currentToOther = if (t.addCurrentStateToOthersOnTransition) Set(current.status.current) else Set()
           val containsIllegalStatuses = current.status.other.intersect(t.illegalStatuses)
           if (containsIllegalStatuses.nonEmpty) {
             val illegalStateTransition = IllegalStatusStateTransition(
               s"Cannot go to $to when article contains $containsIllegalStatuses")
-            return (Failure(illegalStateTransition), SideEffect.fromOutput(Failure(illegalStateTransition)))
+            return (Failure(illegalStateTransition), Set(SideEffect.fromOutput(Failure(illegalStateTransition))))
```

Comment:
Her trenger vi vel egentlig ikke returnere noen sideeffekt lenger?
```suggestion
            return (Failure(illegalStateTransition), Seq.empty)
```